pycrdt_websocket/yroom.py

```diff
@@ -16,6 +16,7 @@
 from anyio.abc import TaskGroup, TaskStatus
 from anyio.streams.memory import MemoryObjectReceiveStream, MemoryObjectSendStream
 from pycrdt import Doc, Subscription
+from tornado.websocket import WebSocketClosedError
```

Comment:
`pycrdt-websocket` is server agnostic, Tornado cannot be used here.

Reply:
:+1: